Cisco/cisco-ise/tests/test_network_info_events.json

```diff
@@ -0,0 +1,33 @@
+{
+  "input": {
+    "sekoiaio": {
+      "intake": {
+        "dialect": "Cisco ISE",
+        "dialect_uuid": "8a9894f8-d7bc-4c06-b96a-8808b3c6cade"
+      }
+    },
+    "message": "INFO: EAP Connection Timeout : Server=servername; NAS IP Address=1.2.3.4; NAS Identifier=A4:57:0:64:47:C0:test1"
+  },
+  "expected": {
+    "message": "INFO: EAP Connection Timeout : Server=servername; NAS IP Address=1.2.3.4; NAS Identifier=A4:57:0:64:47:C0:test1",
+    "event": {
+      "kind": "event",
+      "type": ["info"],
+      "category": "network"
+    },
+    "observer": {
+      "vendor": "Cisco",
+      "product": "Cisco ISE"
+    },
+    "source": {
+      "domain": "servername",
+      "mac": "A4:57:0:64:47:C0:test1",
```

Comment:
This mac address is invalid. Only the first part `A4:57:0:64:47:C0` is the mac address, the second part `test1` seems to be the name of device